src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -84,11 +85,14 @@ def get(self, svc_id):
         return daemon
 
 
-@ApiController('/rgw/proxy')
-@AuthRequired()
+@ApiController('/rgw/proxy', Scope.RGW)
 class RgwProxy(BaseController):
 
     @Proxy()
+    @ReadPermission
+    @UpdatePermission
+    @CreatePermission
+    @DeletePermission
     def __call__(self, path, **params):
```

Comment:
I've created a user with role `Read-Only`, but I'm not able to open the RGW users / buckets pages:

![screenshot from 2018-06-04 18-19-28](https://user-images.githubusercontent.com/14297426/40931697-e8beb35c-6823-11e8-9503-b370b6a10d60.png)

**Possible solution:**
Instead of having a single endpoint to "proxy" all RGW requests, maybe we should have a dedicated endpoint for each RGW request that we need to proxy, so we can control permissions for each endpoint.

Reply:
That's on my todo list.

Reply:
This is #22470 - correct?

Reply:
Yes